tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
     <string name="import_dnscrypt_rules_dialog_message">Please wait… Imported %d rules.</string>
     <string name="import_dnscrypt_rules_complete_dialog_message">Done! Imported %d rules.</string>
     <string name="erase_dnscrypt_rules_dialog_message">Done! The rules are erased.</string>
-    <string name="dnscrypt_many_rules_dialog_message">Too many rules to display. Only the first 1000 is shown.</string>
+    <string name="dnscrypt_many_rules_dialog_message">Too many rules to display. Only the first 1000 are shown.</string>
 
 
-    <string name="pref_tor_virtual_addr_network_IPv4" translatable="false">VirtualAddrNetwork</string>
+    <string name="pref_tor_virtual_addr_network_IPv4" translatable="false">Virtual Address-Network</string>
     <string name="pref_tor_virtual_addr_network_IPv4_summ">When Tor needs to assign a virtual (unused) address because of a MAPADDRESS command from the controller or the AutomapHostsOnResolve feature, Tor picks an unassigned address from this range.</string>
-    <string name="pref_tor_hardware_accel" translatable="false">HardwareAccel</string>
+    <string name="pref_tor_hardware_accel" translatable="false">Hardware Acceleration</string>
     <string name="pref_tor_hardware_accel_summ">Try to use built-in crypto hardware acceleration when available.</string>
-    <string name="pref_tor_avoid_disc_writes" translatable="false">AvoidDiskWrites</string>
-    <string name="pref_tor_avoid_disc_writes_summ">If non-zero, try to write to disk less frequently.</string>
-    <string name="pref_tor_connections_padding" translatable="false">ConnectionPadding</string>
-    <string name="pref_tor_connections_padding_summ">This option governs Tor’s use of padding to defend against some forms of traffic analysis. Disable to save bandwidth.</string>
-    <string name="pref_tor_reduced_connections_padding" translatable="false">ReducedConnectionPadding</string>
-    <string name="pref_tor_reduced_connections_padding_summ">If set enabled, Tor will not hold OR connections open for very long, and will send less padding on these connections. Enable to save bandwidth.</string>
-    <string name="pref_tor_exclude_exit_nodes" translatable="false">ExcludeExitNodes</string>
-    <string name="pref_tor_exclude_exit_nodes_summ">A list of country codes to never use when picking an exit node, that delivers traffic for you outside the Tor network.</string>
-    <string name="pref_tor_exit_nodes" translatable="false">ExitNodes</string>
-    <string name="pref_tor_exit_nodes_summ">A list of country codes to use as exit node - that is, a node that delivers traffic for you outside the Tor network.</string>
-    <string name="pref_tor_exclude_nodes" translatable="false">ExcludeNodes</string>
+    <string name="pref_tor_avoid_disc_writes" translatable="false">Avoid Disk Writes</string>
+    <string name="pref_tor_avoid_disc_writes_summ">Try to write to disk less frequently.</string>
```

Comment:
Is this vm.swappiness? The setting doesn't have a number input in the app, so removed "non-zero".
(The sentence appears to be a double negation with that.)

Reply:
```suggestion
    <string name="pref_tor_avoid_disc_writes_summ">Try to write to internal storage less frequently.</string>
```
Guessing. App storage?